packages/web/src/components/Resource.tsx

```diff
@@ -162,17 +172,19 @@ export const Resource = () => {
           </ListItem>
         )}
         {!!resource.schedule._items.length && (
-          <ListItem component="div">
-            <ListItemIcon classes={listIconClasses}>
-              <ScheduleIcon titleAccess={t("hoursOfOperation")} />
-            </ListItemIcon>
-            <ListItemText>
-              <Typography component="h2" variant="srOnly">
-                {t("hoursOfOperation")}
-              </Typography>
-              <Schedule schedule={resource.schedule} />
-            </ListItemText>
-          </ListItem>
+          <>
```

Comment:
why this change?

Reply:
Ah derp, mistake on my part. [removed](https://github.com/CodeForBoulder/upswyng/pull/479/commits/778e8a84bf58a11eba79bf7fc33699d01f750b91)